plugins.json

```diff
@@ -417,7 +417,7 @@
   },
   {
     "author": "netlify",
-    "description": "Build and deploy Next.js applications with Server-Side Rendering on Netlify. See README for configuration details.",
+    "description": "Build and deploy Next.js applications with server-side rendering and live preview mode. No extra configuration required.",
```

Comment:
@MelanieCrissey not sure we should include the live preview mode piece - could seem like that's just one specific feature we support or something ? 🤷 

fwiw this is the blurb of our plugin repo : )

_This build plugin is a utility for supporting Next.js on Netlify. To enable server-side rendering and other framework-specific features in your Next.js application on Netlify, you will need to install this plugin for your app._

Reply:
I'm down to cut it if you think it's best! I was trying to give this a little more "oompf" and noticed that live preview mode seems like an important feature people are searching for. 

Is your feeling that it's too limiting when included this way? Like, it makes the plugin feel smaller and less functional than it really is?

Reply:
i hear you!!! preview mode was definitely a big ask. but yeah i think it _could_ imply that the plugin only supports SSR and that one feature since we explicitly call it out but there are many other features of Next we support (or have yet to support 😬 in Next 10)

Reply:
Note about other potential things we could add: https://github.com/netlify/netlify-plugin-nextjs/issues/63